CHANGELOG.md

```diff
@@ -8,11 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
- - `Storage::get_dialogue`
+ - `Storage::get_dialogue` to obtain a dialogue indexed by a chat ID.
+ - `RedisStorageError::RowNotFound` to be returned from `RedisStorage::remove_dialogue`.
+ - `InMemStorageError` with a single variant `RowNotFound` to be returned from `InMemStorage::remove_dialogue`.
```

Comment:
I don't really like "`RowNotFound`". Storages are meant to abstract the database layer and "`row`" is the database term. Maybe something like `DialogueNotFound` is better?

And maybe then we also want `get_dialogue` to return `Result<D, Self::Error>>`, instead of `Result<Option<D>, Self::Error>>`?

Reply:
Fixed `RowNotFound` -> `DialogueNotFound`.

Regarding `get_dialogue`: I think that a non-existing dialogue here is not an error. For example, we write `.get_dialogue(chat_id).await.map(Option::unwrap_or_default)` in `DialogueDispatcher`.